spec/nationbuilder_client_spec.rb

```diff
@@ -3,8 +3,8 @@
 describe NationBuilder::Client do
 
   let(:client) do
-    NationBuilder::Client.new('dh',
-                              '6ed3ab9395daf0e3e39098761e85e8e703aa84921c5c1e32637f6984944cf1f2')
+    NationBuilder::Client.new('organizeralexandreschmitt',
+                              '53920a524356034a065515a37650df2bd295971975d5742b9daa50eb8c7404d5')
```

Comment:
Is this a working API key? You should deactivate it once you've generated the fixtures.